automatic/wps-office-free/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,22 @@
+
+  $version = '10.2.0.5820'
+  $regName = $packageName -replace('\-',' ')
+  $regName = $regName -replace('free','*')
+  $registry = Get-UninstallRegistryKey -SoftwareName $regName
+  
+  $packageArgs = @{
+    packageName   = 'wps-office-free'
+    fileType      = 'exe'
+    url           = 'https://jump.wps.com/latest_package?distsrc=00100.00000103&lang=en'
+    silentArgs    = '/S'
+    validExitCodes= @(0)
+    softwareName  = 'WPS Office 2016 Free'
```

Comment:
You're using a different name here than what is passed to the `Get-UninstallRegistryKey`.
What is passed to `Get-UninstallRegistryKey` and the `softwareName` should be the same.